### DISCUSSION TITLE
Bulk editing

### DISCUSSION THREAD

**Discussion by futurewebsites** [area17/twill#1187] [General]
Hi Guys,

I can see the ability to bulk publish / delete in the index view - is it possible to bulk edit other types of columns? i.e lets say an inventory and I want the ability to bulk edit the price column 

many thanks in advance

**Comment by ifox** [comment]
Hi! Not at the moment but this is a great idea, we could leverage the modal after a bulk selection to edit the value of one or more fields.

**Reply by futurewebsites** [reply]
Hi, thats exactly what we are trying to do, it would be nice if we could define editable columns on the controller for the model and related models i.e we have added the twill/meta_data package it would be useful in our case to be able to bulk edit all meta_data for selected products.

We have added a custom index view and included a custom bulk edit vue component but had to jump through hoops to register the vue component whilst we managed to extend the bulk edit dropdown menu its not really brought us any closer to the bulk editing functionality. 

is there a way to extend the core methods registered in main-listing.js?

**Reply by futurewebsites** [reply]
and editable datatabe would be nice something like: https://editor.datatables.net/examples/inline-editing/simple
